Update community fund approval process (#1324)
openjs-foundation/cross-project-council: community-fund/COMMUNITY_FUND_POLICY.md

## CPC Review Schedule & Process 

- CPC will review requests once a month in the CPC private session
- Pending requests will be sent to the private CPC mailing list prior to the review meeting.
  In the context of the lazy concensus process, the pending request list is a proposal to accept
  all of the requests. CPC members should raise any objections before the scheduled meeting,
  otherwise a final decision to approve a request can be confirmed in the meeting.
- The CPC will use a scoring rubric to help with decision making
- If any requests are rejected, an email with the rejected requests will be sent to the private

>   CPC mailing list and CPC members will have 2 days to object before the decision is confirmed.

Reviewer: ```suggestion
  CPC mailing list and CPC members will have 2 days to object before the decision is confirmed.
  Approved requests will be confirmed right away.
```

## CPC Review Schedule & Process 

> - CPC will review requests once a month in the CPC private session

Reviewer: ```suggestion
- CPC will review requests as soon as possible, usually in the next CPC private session (every 2 weeks)
```

- Pending requests will be sent to the private CPC mailing list prior to the review meeting.

>   In the context of the lazy concensus process, the pending request list is a proposal to accept

Reviewer 2: ```suggestion
  In the context of the lazy consensus process, the pending request list is a proposal to accept
```
